fastf1/utils.py

```diff
@@ -204,7 +204,7 @@ def to_datetime(x: Union[str, datetime.datetime]) \
     Args:
         x: timestamp
     """
-    if isinstance(x, str):
+    if isinstance(x, str) and x.strip():
```

Comment:
I'm not really happy with the usage of  ``.strip()`` here. It would remove an arbitrary number of leading and trailing spaces. I suppose the intention is to check if a string is empty, where any string that only contains spaces is also considered as empty. But really, why should a string that contains only spaces be treated differently from a string that contains any other character? Both are invalid timestamps.
I'm happy to treat a truly empty string differently because that can be a reasonable string NaT value. But as soon as the string contains any characters, and it cannot be parsed, this should be treated as an error and logged.

Reply:
Yes, you are right. Removing the `.strip()` would than only check for empty strings, and strings containing only spaces would throw an exception. Or do you have a better alternative? An other way would be to check if the string only contains spaces before the try block, but i don't like that approach.

Reply:
No, I think that's the way to go.

Reply:
Removed `.strip()` now.